os/include/tinyara/binary_manager.h

```diff
@@ -71,6 +71,18 @@ enum binmgr_partition_type {
 	BINMGR_PART_MAX,
 };
 
+/* Types of resources which use kernel resource, inode */
+enum binmgr_resource_type {
+	BIN_RESOURCE_MQ = 0,
```

Comment:
You'd better check CONFIG_DISABLE_MQUEUE also.

Reply:
Binary manager has a dependency of CONFIG_DISABLE_MQUEUE.
So we don't need to add it here I think.